core/ui/compose/designsystem/src/main/res/values/strings.xml

```diff
@@ -5,4 +5,6 @@
     <string name="designsystem_molecule_email_address_input_label">Email address</string>
     <string name="designsystem_molecule_password_input_label">Password</string>
     <string name="designsystem_molecule_error_view_button_retry">Retry</string>
+    <string name="designsystem_molecule_a11y_checked">%1$s, checked</string>
+    <string name="designsystem_molecule_a11y_unchecked">%1$s, not checked</string>
```

Comment:
I think we can increase the accessibility of our code by avoiding abbreviations whenever possible :smile: 

In similar cases we are using the term "content description". Maybe `designsystem_molecule_checkbox_content_description_checked` and `designsystem_molecule_checkbox_content_description_unchecked`? (If we need this at all)